client/state/data-layer/wpcom/concierge/shifts/index.js

```diff
@@ -0,0 +1,50 @@
+/** @format */
```

Comment:
this may seem quite obtuse, but normally we would expect to find this file at `…/wpcom/concierge/schedules/shifts` because of the actual API path it calls. The idea is that we can work background from network call to calling code.

Reply:
That's reasonable. I've moved them in c12feb2.